python/samples/04.ai.a.teamsChefBot/src/app.py

```diff
@@ -0,0 +1,72 @@
+"""
+Copyright (c) Microsoft Corporation. All rights reserved.
+Licensed under the MIT License.
+"""
+
+from typing import TypeVar
+
+import uvicorn
+from botbuilder.core import (BotFrameworkAdapterSettings, MemoryStorage,
+                             TurnContext)
+from botbuilder.schema import Activity
+from fastapi import FastAPI, Request, Response
+from teams import (AIHistoryOptions, AIOptions, Application,
+                   ApplicationOptions, AzureOpenAIPlanner,
+                   AzureOpenAIPlannerOptions, ConversationState, TempState,
+                   TurnState, UserState)
+
+from src.config import Config
+
+# Initialize Teams AI application
+StateT = TypeVar("StateT", bound=TurnState[ConversationState, UserState, TempState])
```

Comment:
this line is not necessary, if someone is not using a custom turn state they should use `TurnState` class directly.

Reply:
merged your changes to `TurnState` and updated the code